Yoast/yoast-components
# Add 2px padding to svg angles in analysisCollapsible

## Summary

This PR can be summarized in the following changelog entry:

* Added bottom padding to svg angles in analysisCollapsible component to center them.


@hedgefield requested to add padding to the svg angles in analysisCollapsible to center them.

_For the people with ultra sight:_
Before:
<img width="176" alt="schermafbeelding 2018-04-09 om 15 57 21" src="https://user-images.githubusercontent.com/17744553/38502077-cc3601b8-3c0e-11e8-86c1-b61a50694395.png">

After:
<img width="163" alt="schermafbeelding 2018-04-09 om 15 57 27" src="https://user-images.githubusercontent.com/17744553/38502082-cf1e3c06-3c0e-11e8-8437-c4e9f27819ea.png">

## Test instructions

This PR can be tested by following these steps:

* Inspect the angles and see the 2px bottom padding on the angles.
